README.md

```diff
@@ -52,6 +52,10 @@ You can use one of our precompiled [binaries](https://github.com/herber/cargo/re
     <td class="shortcut">alt</td>
     <td>Open menu</td>
   </tr>
+  <tr>
+    <td class="shortcut">ctrl + enter</td>
```

Comment:
What about macOS?

Reply:
I followed the Chrome implementation which uses Ctrl+Enter on both Windows and macOS.

Reply:
@justinhhorner Could you add `command + enter` to the docs.

Reply:
We should follow Chrome's implementation, so there is no need to do anything. Thanks again for the clarification.